app/services/arcgis_api/geocoder.rb

```diff
@@ -0,0 +1,58 @@
+module ArcgisApi
+  class Geocoder
+    def suggest(text)
```

Comment:
I recommend marking the public functions and maybe the class up with some [YARD](https://rubydoc.info/gems/yard/file/docs/GettingStarted.md) / [RDoc](https://devhints.io/rdoc) documentation, i.e. esp. since this isn't a precise Ruby interface for the ArcGIS API. You could also rename this to `suggest_address` or similar to help clarify the difference.